package/libyui.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Sep 23 12:44:47 UTC 2019 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Added CurrentItem property to new ItemSelector widget for consistency
+  ((bsc#1084674)
```

Comment:
Mismatched parentheses leave you trapped in someone's dream.[)](https://en.wikipedia.org/wiki/Inception)

```suggestion
  (bsc#1084674)
```

Reply:
OMG, we can't have people get caught into others' dreams. ;-)
Fixed.